torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -526,20 +558,77 @@ class ScriptModuleSerializer {
     }
   }
 
-  void writeByteCode(const Module& module, bool save_mobile_debug_info) {
+  void backPortByteCode(const uint64_t version) {
```

Comment:
Note that this is little different then what has been suggested? At least thats what I think.
I think backport would take existing model and backport. So you would have saved the model in latest format and then backport.
I think at API level this change is ok probably, that is assuming we want to expose version number in save_for_lite_interpreter. I presume this will be fixed later so that it becomes actual backport function.